### ISSUE TITLE
Refresh remote branches automatically

### ISSUE THREAD

**Issue by Tvax** [gitahead/gitahead#436]
Remote branches should be kept up to date with the repo at any time, so Gitahead should refresh them from the repo each time the user pull or fetch a git repository by running this command:
`git remote update origin --prune`

**Comment by MJBrune** [comment]
Yes please! This is already exists in the UI as a button under the repository chooser.

**Comment by ZaLiTHkA** [comment]
is this not already covered by the app settings?

![Screenshot from 2020-09-03 09-42-56](https://user-images.githubusercontent.com/2149312/92086221-2baa9e00-edca-11ea-8a2a-0b6ded36c4fe.png)

**Comment by Tvax** [comment]
Indeed never been aware of that option, checking `Prune when fetching` solves my issue.
Thank you @ZaLiTHkA

**Comment by ZaLiTHkA** [comment]
@Tvax no worries. I only found GitAhead the other day, so (as one does with a new toy) I went looking through all the options, that one came to mind when I saw the title of this issue. (:
